qcodes/tests/test_field_vector.py

```diff
@@ -82,3 +82,26 @@ def test_cylindrical_properties(cylindrical0):
     cartisian1 = FieldVector(**cylindrical1).get_components("x", "y", "z")
 
     assert np.allclose(cartisian0, cartisian1 * np.array([-1, -1, 1]))
+
+@given(
+    random_coordinates["cylindrical"],
+    random_coordinates["spherical"]
+)
+@settings(max_examples=10)
+def test_triangle_inequality(cylindrical0, spherical0):
+    cylindrical0 = FieldVector(**dict(zip(["rho", "phi", "z"], cylindrical0)))
+    spherical0 = FieldVector(**dict(zip(["r", "phi", "theta"], spherical0)))
+
+    assert (cylindrical0 + spherical0).norm() <= (cylindrical0.norm() + spherical0.norm())
+    assert cylindrical0.distance(spherical0)) <= (cylindrical0.norm() + spherical0.norm())
```

Comment:
mypy found invalid syntax :)
```suggestion
    assert cylindrical0.distance(spherical0) <= (cylindrical0.norm() + spherical0.norm())
```